src/main/java/converter/ConstantRateProvider.java

```diff
@@ -0,0 +1,21 @@
+package converter;
+
+import converter.interfaces.IRateProvider;
+
+import java.math.BigDecimal;
+
+/**
+ * There are no requirements for RateProvider implementation
+ * but interfaces without the implementation are considered a code smell sometimes
+ */
+public class ConstantRateProvider implements IRateProvider {
+    @Override
+    public BigDecimal getRateUsdToEur() {
```

Comment:
I don't think soma APIs will return exchange rate as BigDecimal, but let's consider this later :)